.codex/skills/mgsc-table-probe/scripts/mgsc_table_probe.py

```
@@ -0,0 +1,589 @@
+        cases.append(
+            Case(
+                name=stem,
+                bin_path=bin_path,
+                disasm_path=disasm if disasm.exists() else None,
+                src_path=src if src.exists() else None,
+            )
+        )
+    return cases
+
+
+def maybe_copy_to_tmp(case: Case, run_dir: Path) -> Path:
```

Ruff (0.15.2)

[warning] Unused function argument: `run_dir`

(ARG001)